modules/simple_forms_api/app/models/simple_forms_api/vba_26_4555.rb

```diff
@@ -12,15 +12,15 @@ def initialize(data)
 
     def as_payload
       {
-        remarks: data['remarks'],
-        otherConditions: data['other_conditions'],
+        remarks: @data['remarks'],
+        otherConditions: @data['other_conditions'],
         livingSituation: living_situation_payload,
         previousHiApplication: previous_hi_application_payload,
         previousSahApplication: previous_sah_application_payload,
         veteran: veteran_payload,
-        statementOfTruthSignature: data['statement_of_truth_signature'],
-        statementOfTruthCertified: data['statement_of_truth_certified'],
-        formNumber: data['form_number']
+        statementOfTruthSignature: @data['statement_of_truth_signature'],
+        statementOfTruthCertified: @data['statement_of_truth_certified'],
+        formNumber: @data['form_number']
       }
```

Comment:
Adding `attr_accessor :data` at the top of the file might make this change unnecessary